### ISSUE TITLE
How can the Arduino tell whether there is a motor connected or not, and how ill the GUI know this?

### ISSUE THREAD

**Issue by sbuckleybonanno** [sgpthomas/labsight#18]

**Comment by sgpthomas** [comment]
I don't think that there is a way that the arduino can detect if a motor is attached. I guess if we can get the encoders working, the arduino can spin the supposed motor and if there is a response from the encoder, we can assume that a motor is attached. We can implement a command in the protocol for this. Maybe `check` and it returns the number of motors attached?
